src/components/Prompt/messages.js

```diff
@@ -27,6 +27,11 @@ const messages = defineMessages({
     description: 'Done step prompt header',
     id: 'frontend-app-ora.Prompt.header.done',
   },
+  promptTitle: {
```

Comment:
not sure what I think about this as a default value but I also don't think it really matters